mmv1/third_party/terraform/services/accesscontextmanager/resource_access_context_manager_service_perimeter_dry_run_egress_policy_test.go

```diff
@@ -0,0 +1,156 @@
+package accesscontextmanager_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
```

Comment:
The import issues should be due to this line, after this PR: https://github.com/GoogleCloudPlatform/magic-modules/pull/10339/files (other files/lines may have the same issue, I didn't check)

```suggestion
	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
```

Reply:
Thanks for pointing this out!